components_esp32.lock

```diff
@@ -0,0 +1,21 @@
+dependencies:
```

Comment:
we probably also want to add the component lock file to the .gitignore list

Reply:
Yeah that's a good idea

Reply:
Done